src/pages/Community/PostDetail.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import PostNav from './components/PostNav';
+import PostDetailBox from './components/PostDetailBox';
+import PostUser from './components/PostUser';
+import PostTrainer from './components/PostTrainer';
+import './PostDetail.scss';
+
+const PostDetail = () => {
+  const [communityData, setCommunityData] = useState([]);
+  const { id } = useParams();
+
+  useEffect(() => {
+    fetch('/data/communityData.json')
+      .then(response => response.json())
+      .then(data => setCommunityData(data));
+  }, []);
+
+  const post = communityData.find(post => post.id === parseInt(id, 10));
+
+  return (
+    <div className="postDetail">
+      <div className="container">
+        <PostNav text="   " />
+        {post ? <PostDetailBox post={post} /> : <p>Loading...</p>}
```

Comment:
존재하지 않는 경우에 Loading이라고 보여주는 건 어색할 것 같네요!

Reply:
그리고 해당 컴포넌트를 보니 post 객체를 구조분해할당 하셨는데, 여기서 이렇게 전달할 수도 있습니다.
적용해보시고 콘솔 찍어서 확인해보세요!
```suggestion
        {post ? <PostDetailBox {...post} /> : <p>Loading...</p>}
```